front/app/containers/Admin/users/UserTableActions.tsx

```diff
@@ -326,98 +323,130 @@ const UserTableActions = ({
+                      <DropdownList>
+                        {manualGroups.data.map((group) => (
+                          <DropdownListItem
+                            key={group.id}
+                            onClick={toggleGroup(group.id)}
+                            className="e2e-dropdown-item"
+                          >
+                            <DropdownListItemText>
+                              <T value={group.attributes.title_multiloc} />
+                            </DropdownListItemText>
+                            <Checkbox
+                              checked={includes(selectedGroupIds, group.id)}
+                              onChange={toggleGroup(group.id)}
+                            />
+                          </DropdownListItem>
+                        ))}
+                      </DropdownList>
+                    }
+                    footer={
+                      <DropdownFooterButton
+                        className="e2e-dropdown-submit"
+                        buttonStyle="cl-blue"
+                        onClick={addUsersToGroups}
+                        processing={processing}
+                        fullWidth={true}
+                        padding="12px"
+                        whiteSpace="normal"
+                        disabled={
+                          !selectedGroupIds || selectedGroupIds.length === 0
+                        }
+                      >
+                        <FormattedMessage {...messages.moveUsersButton} />
+                      </DropdownFooterButton>
+                    }
+                  />
+                </ActionButtonWrapper>
+              )}
+
+              {groupType === 'manual' && selectedUsers !== 'none' && (
+                <Button
+                  onClick={handleGroupsDeleteClick}
+                  className="hasLeftMargin"
+                  buttonStyle="admin-dark-text"
+                >
+                  <StyledIcon name="delete" />
+                  <FormattedMessage {...messages.membershipDelete} />
+                </Button>
+              )}
+
+              <Button
+                onClick={exportUsers}
+                className={`export e2e-${exportType} hasLeftMargin`}
+                buttonStyle="admin-dark-text"
+              >
+                <StyledIcon name="user-data" />
+                <FormattedMessage {...messages[exportType]} />
+              </Button>
+            </ActionButtons>
+          </>
         )}
-
-        <Button
-          onClick={exportUsers}
-          className={`export e2e-${exportType} hasLeftMargin`}
-          buttonStyle="admin-dark-text"
-        >
-          <StyledIcon name="user-data" />
-          <FormattedMessage {...messages[exportType]} />
-        </Button>
-      </ActionButtons>
-    </TableOptions>
+      </StyledBox>
+      <Box flex="0 0 250px">
+        <SearchInput
+          onChange={onSearch}
+          // Not important here. Requires quite some refactoring
```

Comment:
Isn't it just adding one more pro like `usersDataLength: number` now? It seems it was harder when it was in `UsersGroupHeader`.

Reply:
You're right. I had missed it. Added now